.env

```diff
@@ -15,6 +15,7 @@ PROMETHEUS_VERSION=2.12.0
 
 # NeoFS images versions
 NEOFS_IMAGES_VERSION=0.0.5
+NEOFS_DROPUI_VERSION=0.0.2-pre
```

Comment:
I would prefer to have this change in separate commit since it's not related to #2

Reply:
done